### PR TITLE
Motivational model

### PR DESCRIPTION
**Pull request by lollyluan**
Update the documentation of Who/Do/Be/Feel and diagram of motivational model

### CODE REVIEW

File: requirements/motivational_model.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+**Game setup and management**
+- Host a game
+- Customize the timer for the game
+- Set number of players
+- Assign roles
+- Assign groups
+- Control the process (e.g., end early, add time, return to a previous step)
+
+**Player interaction and participation**
+- Scan the QR code/room number to join the game
```
Comment (JiayuanLi123) [inline requirements/motivational_model.md:29]: maybe can add "Present arguments publicly to sway group opinion"

**Review by JiayuanLi123 — COMMENTED** [review]
The comment is just a suggestion

**Review by six-chen — COMMENTED** [review]
all good

**Comment by RunbingWANG** [comment]
It's great.

**Pull request merged**